python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -491,7 +491,7 @@ def write_parquet(
             "Valid values are '1.0' and '2.0'"
         )
 
-    dict_policy = (
+    cdef cudf_io_types.dictionary_policy dict_policy = (
```

Comment:
There's a couple of missing/wrong types, so when I migrated all the I/O enums to use cpdef, there were a couple of other files that needed to be adjusted.